src/custom_nodes/common/buffersqueue.cpp

```diff
@@ -47,8 +50,17 @@ bool BuffersQueue::returnBuffer(void* buffer) {
     returnStream(getBufferId(buffer));
     return true;
 }
+
 int BuffersQueue::getBufferId(void* buffer) {
     return (static_cast<char*>(buffer) - memoryPool.get()) / singleBufferSize;
 }
+
+size_t BuffersQueue::getSize() {
```

Comment:
Is this number of buffers or total number of bytes allocated?

Reply:
size(singleBufferSize * streamsLength) so total number of bytes allocated